src/components/Stage.js

```diff
@@ -247,56 +231,14 @@ class Stage extends Component {
             textOverflow: "ellipsis",
           }}
         >
-          <div

```

Comment:
If you remove the fought-for pencils (https://octopus-tsk9219.slack.com/archives/CJVSM25QR/p1559756821001900), I would suggest moving the plus to the same place where the pencil was (otherwise it is partially covered by the number of publications label) - or just keeping the pencils but giving them the logic of the plus you added.